<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Education</class>
 <widget class="QMainWindow" name="Education">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>927</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #B3FFFF, stop:1 #99B3CC);</string>
   </property>
   <widget class="QStackedWidget" name="which_place">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>1000</width>
      <height>618</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="PCsignwindow">
     <property name="styleSheet">
      <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.495 rgba(255, 255, 255, 255), stop:0.505 rgba(255, 0, 0, 255), stop:1 rgba(255, 0, 0, 255));
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #B3FFFF, stop:1 #99B3CC);</string>
     </property>
     <widget class="QLabel" name="pc_picture_sign">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>160</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: #fff;
    border-radius: 100%;
}</string>
      </property>
      <property name="text">
       <string>摄像头拍摄人脸位置</string>
      </property>
     </widget>
     <widget class="QLabel" name="time_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>0</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black; 
border-radius: 1px; border: 1px solid black; padding: 2px;
font: 12pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>签到系统</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>311</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="id_card_3">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 14pt &quot;楷体&quot;;</string>
           </property>
           <property name="text">
            <string>一卡通账号：</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_account_3"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="id_password">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 14pt &quot;楷体&quot;;</string>
           </property>
           <property name="text">
            <string>一卡通密码：</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_password"/>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>签到</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="PCexamwindow"/>
    <widget class="QWidget" name="PCclasswindow"/>
    <widget class="QWidget" name="hisconsignwindow"/>
    <widget class="QWidget" name="hisconexamwindow"/>
    <widget class="QWidget" name="hisconclasswindow">
     <property name="styleSheet">
      <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.495 rgba(255, 255, 255, 255), stop:0.505 rgba(255, 0, 0, 255), stop:1 rgba(255, 0, 0, 255));
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #B3FFFF, stop:1 #99B3CC);</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>301</width>
        <height>361</height>
       </rect>
      </property>
      <property name="text">
       <string>你好</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>221</width>
        <height>331</height>
       </rect>
      </property>
      <property name="text">
       <string>我不好</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>本机打卡人员：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>C02浓度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>环境温度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>环境温度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>环境湿度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>400</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>环境湿度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>光照：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>450</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>光照：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>噪声：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>490</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>噪声：</string>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #7f7f7f, stop:1 #000000);
border-radius: 1px; border: 1px solid black; padding: 2px;
font: 12pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>当前时间：</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_time">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>30</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 14pt &quot;AcadEref&quot;;
background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #7f7f7f, stop:1 #000000);</string>
    </property>
    <property name="digitCount">
     <number>20</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>教育陪伴系统</string>
    </property>
    <addaction name="actionexit"/>
    <addaction name="actionexit_2"/>
   </widget>
   <widget class="QMenu" name="menu_PC">
    <property name="title">
     <string>纯PC界面</string>
    </property>
    <addaction name="action_sign_pc"/>
    <addaction name="action_class_pc"/>
    <addaction name="action_exam_pc"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>开发板端</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_sign_hiscon"/>
    <addaction name="action_class_hiscon"/>
    <addaction name="action_exam_hiscon"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_PC"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionexit">
   <property name="text">
    <string>run</string>
   </property>
  </action>
  <action name="action_sign_hiscon">
   <property name="text">
    <string>sign in</string>
   </property>
  </action>
  <action name="action_class_hiscon">
   <property name="text">
    <string>in class</string>
   </property>
  </action>
  <action name="action_sign_pc">
   <property name="text">
    <string>sign in</string>
   </property>
  </action>
  <action name="action_class_pc">
   <property name="text">
    <string>in class</string>
   </property>
  </action>
  <action name="action_exam_pc">
   <property name="text">
    <string>exam</string>
   </property>
  </action>
  <action name="action_exam_hiscon">
   <property name="text">
    <string>in exam</string>
   </property>
  </action>
  <action name="actionexit_2">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
